### PR TITLE
Feat/eleven labs

### PR DESCRIPTION
**Pull request by zbigniewsobiecki**
* brings support of eleven labs text-to-speech
* tests for services
* simplified integration code
* support for custom integration options (like "voice")

### CODE REVIEW

File: prr/services/providers/elevenlabs/tts.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from elevenlabs import generate, set_api_key
```
Comment (meal) [inline prr/services/providers/elevenlabs/tts.py:1]: there's `elevenlabs` package missing in pyproject.toml,
you can add it with `poetry add elevenlabs`

File: examples/text-to-speech/hello.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+version: 1
+prompt:
+  content: "Good Morning, Warsaw!"
+services:
+  models:
+    - 'elevenlabs/tts/eleven_monolingual_v1'
+  options:
```
Comment (meal) [inline examples/text-to-speech/hello.yaml:7]: I'd mention somewhere that you can also use your own voice id (the hash) for voice

File: prr/services/providers/elevenlabs/tts.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    options = ["voice"]
+
+    def run(self):
+        audio = generate(
+            text=self.request.prompt_content,
+            voice=self.option("voice"),
+            model=self.service_config.model_name(),
+        )
+
+        self.response = ServiceResponse(audio)
```
Comment (meal) [inline prr/services/providers/elevenlabs/tts.py:25]: right now what we have is only the length of response, without actually storing/showing the response.

```python
import os
path = "./audio.mp3"
f = open(path, 'wb)
f.write(audio)
```
would also save the audio result to the mp3 file

**Review by meal — CHANGES_REQUESTED** [review]
good work on refactoring, just minor fixes needed for elevenlabs integration

File: prr/services/providers/elevenlabs/tts.py (hunk 3 above)
Comment (zbigniewsobiecki) [reply prr/services/providers/elevenlabs/tts.py:25]: The file gets saved with using -l option as well as you can use -q to pipe it to mp3 file.

**Review by meal — APPROVED** [review]
lgtm now

**Pull request merged**